packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/datacenters.html

```diff
@@ -13,15 +13,8 @@ <h2 data-translate="dedicatedCloud_datacenters_title"></h2>
         data-ng-if="$ctrl.displayMigrationBanner()"
```

Comment:
you can use one time binding `::` or use directly the properties `migrationBannerAvailable` and `hasLegacyDatacenter`. The function here is not very useful as the values can't change